src/main/scala/org/ergoplatform/nodeView/wallet/persistence/OffChainRegistry.scala

```diff
@@ -0,0 +1,67 @@
+package org.ergoplatform.nodeView.wallet.persistence
+
+import org.ergoplatform.nodeView.history.ErgoHistory
+import org.ergoplatform.wallet.boxes.TrackedBox
+
+/**
+  * Holds version-agnostic indexes (such as off-chain boxes) in runtime memory.
+  */
+final case class OffChainRegistry(height: Int,
```

Comment:
please add parameters annotations.
E.g. it is not clear - if I have an onchain box `A` and put a transaction that spend box `A` and creates a new one box `B` - `onChainBalances` contains box `A` and  `offChainBalances` contains box `B`? Or box `A` is removed from `onChainBalances`?

Reply:
The latter.